build-scripts/warnings.cmake

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# global additional warnings. Keep those options consistent with wamr-compiler/CMakeLists.txt.
```

Comment:
i don't understand this comment. wamr-compiler/CMakeLists.txt includes this file, doesn't it?